spring-auto-restdocs-core/pom.xml

```diff
@@ -78,6 +78,10 @@
             <artifactId>javax.el-api</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.glassfish.web</groupId>
+            <artifactId>javax.el</artifactId>
```

Comment:
We had the el api as a dependency, but no implementation for it. Hibernate Validator started enforcing an implementation, see https://hibernate.atlassian.net/browse/HV-1131 and http://hibernate.org/validator/documentation/getting-started/